radsat-sk/operation/crypt/RCrypt.c

```diff
@@ -0,0 +1,66 @@
+#include "RCrypt.h"
+
+int getPaddedBufSize(int plaintextLen);
+void padBuffer(uint8_t* buff, uint8_t* paddedBuff, int buffLen, int newBuffLen);
+void encrypt(uint8_t* plaintext, int plaintextLen);
+size_t decrypt(uint8_t* ciphertext, int ciphertextlen);
+
+
+
+/**
+ * @brief Gets the next multiple of 16 from the size of the input buffer
+ * @header "operations/crypt/encrypt.h"
+ * @param plaintextLen - The size of the unpadded buffer containing data to be encrypted.
```

Comment:
Don't use hypens in the Doxygen comment blocks. Just use a single space - the doxygen formatter will figure the rest out if we generate html docs from these (which I assume we will eventually)